optuna/samplers/_cmaes.py

```diff
@@ -142,6 +150,16 @@ def objective(trial):
                 to set this flag :obj:`True` when the :class:`~optuna.pruners.HyperbandPruner` is
                 used. Please see `the benchmark result
                 <https://github.com/optuna/optuna/pull/1229>`_ for the details.
+
+        use_separable_cma:
+            If this is :obj:`True`, the covariance matrix is constrained to be diagonal.
+            Due to reduce the model complexity, the learning rate for the covariance matrix
+            is reduced. Consequently, this algorithm outperforms CMA-ES on separable functions.
```

Comment:
The abstract of the [paper](https://hal.inria.fr/inria-00287367/document) says
> Because the model complexity is reduced, the learning rate for the covariance matrix can be increased.

I guess it is not consistent with the second sentence of this description.